crates/bin/src/args.rs

```diff
@@ -218,6 +219,10 @@ pub struct Args {
     #[clap(help_heading = "Options", long)]
     pub json_output: bool,
 
+    /// Provide the github token for accessing the restful API of api.github.com
+    #[clap(help_heading = "Options", long, env = "GITHUB_TOKEN")]
+    pub github_token: Option<CompactString>,
```

Comment:
@passcod I'm wondering if we should provide a few default `GITHUB_TOKEN`s for our user that is read-only fo annoymous GitHub user accounts (could be stored in our repository) and then randomly picked from one of them just so that they won't get rate limited by GitHub.

GitHub put a very tight rate limit for unauthorized requests:
60 requests per hour per origin IP address

That's way too limited.

Reply:
But providing default github tokens does have a few security implications and I would rather not to.

For GHA, they could just provide their token to speedup the operation.

@taiki-e Is it a good idea to add a GITHUB_TOKEN parameter to https://github.com/taiki-e/install-action ?

Reply:
yeah including tokens is a bad idea. that ratelimit isn't so bad. i think it allows installing 30+ things with this feature per hour? that's not nothing